homeassistant/components/asuswrt/sensor.py

```diff
@@ -67,6 +72,21 @@
         SENSOR_FACTOR: 1000000000,
         SENSOR_ICON: "mdi:upload",
     },
+    SENSORS_LOAD_AVG[0]: {
+        SENSOR_NAME: "Load Avg (1m)",
+        SENSOR_UNIT: PERCENTAGE,
```

Comment:
Are these percentages or Linux load values?

Reply:
This are equal to the value provided in the AsusWrt UI. I assumed it was percentages, but may be I'm wrong, also because I don't know what exactly are `Linux load values`.

Reply:
In general load averages are not expressed in %, looking at the values in the test, they are loads, not %.

https://www.howtogeek.com/194642/understanding-the-load-average-on-linux-and-other-unix-like-systems/

Reply:
I should use in any case a unit of measure to have the values represented as graph in HA. I do not found a standard UOM for this, should I define a new one?

Reply:
It doesn't have a unit

Reply:
What I would say is that if we don't assign a UOM to these sensors, in the history we will have something like this:

![image](https://user-images.githubusercontent.com/60491700/123692039-a1e72800-d856-11eb-86d4-0bd3c084f830.png)

instead of a graphic like this:

![image](https://user-images.githubusercontent.com/60491700/123692361-ff7b7480-d856-11eb-8b90-390af2a370d6.png)

I think that a graphic in this case should be preferable.

Reply:
Unitless states do not have graphs.
And loads do not have units.

We should (and will not) make backend workarounds for a frontend representation issue.

Reply:
Ok, will fix accordingly.